apiserver/facades/agent/provisioner/provisioninginfo.go

```diff
@@ -29,15 +29,15 @@ import (
 )
 
 // ProvisioningInfo returns the provisioning information for each given machine entity.
-func (p *ProvisionerAPI) ProvisioningInfo(args params.Entities) (params.ProvisioningInfoResults, error) {
+func (api *ProvisionerAPI) ProvisioningInfo(args params.Entities) (params.ProvisioningInfoResults, error) {
```

Comment:
All changes to this file were receiver renaming is can be ignored by reviewer(s).